lib/grid/index.js

```diff
@@ -0,0 +1,31 @@
+import angular from 'angular';
+import css from './grid.css!'
+
+var grid = angular.module('grid', []);
+
+// COLUMN
+// ------------------------
+grid.directive('column', function column(){
```

Comment:
Can we make all custom element directives include a `-` for forward consistency with web components? Otherwise it won't be possible to replace these angular directives with other web components. A hyphen also helps identify native vs custom elements (this is why they made this requirement for WC).

The [recommendation](http://webcomponents.org/articles/how-should-i-name-my-element/) is to use a prefix to namespace elements (e.g. `polymer-something`); should we use `gu-`?